include/nbl/builtin/glsl/material_compiler/common.glsl

```diff
@@ -565,6 +565,46 @@ nbl_glsl_MC_eval_pdf_aov_t nbl_glsl_MC_instr_execute_BLEND(
 	return retval;
 }
 
+// vec3 nbl_glsl_MC_oriented_material_t_getEmissive(in nbl_glsl_MC_oriented_material_t orientedMaterial)
+// {
+// 	if (orientedMaterial.emitter_id == NBL_GLSL_MC_INVALID_EMITTER_ID) {
+// 		return vec3(0.0);
+// 	}
+// 	nbl_glsl_MC_emitter_t emitter = nbl_glsl_MC_fetchEmitterData(orientedMaterial.emitter_id - 1);
+// 	vec3 emissive = nbl_glsl_decodeRGB19E7(emitter.emissive);
+// #if _NBL_VT_FLOAT_VIEWS_COUNT
+// 	if (emitter.normalizeEnergy != 0.0f) {
+//  		return emissive * nbl_glsl_vTextureGrad(emitter.emissionProfile, vec2(0.5), mat2(0.0)).r;
+// 	}
+// #endif
+// 	return emissive;
+// 	// if (nbl_glsl_unpackWrapModes(orientedMaterial.texID) == uvec2(3,3)) {
+// 	// 	return nbl_glsl_decodeRGB19E7(orientedMaterial.emissive);
+// 	// }
+// 	// return nbl_glsl_vTextureGrad(orientedMaterial.texID.xy, vec2(0.5), mat2(1.0)).r * vec3(2.0);
+// }
+
+vec2 nbl_glsl_MC_convert_dir_to_ies_uv(vec3 dir) {
+	return vec2((atan(dir.x, dir.y) + nbl_glsl_PI) / (2.0*nbl_glsl_PI), acos(dir.z) / nbl_glsl_PI);
+}
```

Comment:
why did you choose this mapping?

Reply:
also this needs to go to some `nbl/builtin/glsl/ies` just how our envmap sampling and sphere sampling did

Reply:
you know the GPU hates inverse trig already, use octahedral  instead plz

Reply:
I'd take an extra 3 bit enum about the domain size
```cpp
MIRROR_180_BITS = 0b001, // Last Angle is 180, so map V slightly differently
MIRROR_90_BITS = 0b010, // Last Angle is 90, so map both U and V slightly differently
ISOTROPIC_BITS = 0b011, // texture to sample is Nx1, pretend v=middle always  
FULL_THETA_BIT = 0b100, // handle extended domain and rotate by 45 degrees for anisotropic
```